core/models.py

```
@@ -912,7 +912,28 @@ def post_title(self):
+                    project_id=self.project.id if self.project else None,
+                    blog_post_id=self.id,
+                    error=error_message,
+                )
+                return False
+        except ValueError as e:
+            logger.error(
+                "[Submit Blog Post] Validation error, deleting.",
+                project_id=self.project.id if self.project else None,
+                blog_post_id=self.id,
+                error=str(e),
+            )
```

Ruff (0.13.3)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)